src/com/ppm/integration/agilesdk/connector/agilecentral/RallyIntegrationConnector.java

```diff
@@ -26,13 +23,13 @@ public String getTargetApplicationVersion() {
 	}
 
 	@Override
 	public List<Field> getDriverConfigurationFields() {
 		return Arrays.asList(new Field[] {
-			new PlainText(Constants.KEY_BASE_URL, "BASE_URL", "https://rally1.rallydev.com", true),
+            new PlainText(Constants.KEY_BASE_URL, "BASE_URL", "https://demo-rally.rallydev.com", true),
```

Comment:
Do you really have to set a default value in this field? Can't we let customers write their own instance URL?